### DISCUSSION TITLE
Note Velocity easier to work with

### DISCUSSION THREAD

**Discussion by o010b** [SynthstromAudible/DelugeFirmware#2246] [Ideas]
Working with note velocity becomes tedious fast when there is no overview and you need to press and hold each note, nudge the knob to see the value, spin spin spin to set it to desired value, press and hold next, nudge the know and so on. 

I have two suggestions related to this; 

1. In Clip Automation View, select the modulation button for "Velocity". Deluge displays columns for velocities where there are notes. Edit like other parameters in Clip Automation. (Maybe add aftertouch as well?)
2. In Clip Mode, press and hold an existing note. Velocity is displayed as a column over mute buttons (in an unused color from mute functionality). If holding a button in the note column it would be compatible with existing functionality to manipulate rows. 

The only problem I can see immediately is in method no.1 with multiple notes in the same sequencer column. Maybe there is a clever way to cycle between them. Otherwise, at least for starters, I think it would be easy enough to edit all of them simultaneously in automation view and adjust individual notes with method no.2 if needed.

**Comment by sapphire-arches** [comment]
(1) has been implemented, check out the latest nightly build =)

**Reply by o010b** [reply]
Yay, neato!
